src/languages/asciidoc.js

```diff
@@ -110,16 +110,16 @@ export default function(hljs) {
       // inline unconstrained strong
       {
         className: 'strong',
-        begin: /\*{2}/,
-        end: /(\n{2}|\*{2})/
+        begin: /\*{2}[\w\s\n]+/,
+        end: /\*{2}/
       },
       // inline constrained strong
       {
         className: 'strong',
         // must not follow a word character or be followed by an asterisk or space
         // must not be preceded by the escape character \
-        begin: '(?<!\\\\)\\B\\*(?![\\*\\s])',
-        end: '(\\n{2}|\\*)',
+        begin: /(?<!\\)\B\*\w/,
+        end: /\*/,
```

Comment:
Nope, can't use look-behind, not supported by all browsers yet. Sorry.  You could look at making this a chain (with `starts`) or rebase onto #2834 and use the syntactic sugar to the same end.  But none of this is true look-behind... we can only look-forward at this time.  So in order to do something like this you'll need to consume the prior character (ie. another mode can't have already consumed it).

Reply:
> or rebase onto #2834 and use the syntactic sugar to the same end.

That's what I'd recommend if you know how.

Reply:
Or like I suggested earlier capture the `\*` pattern as it's own previous rule (to prevent this rule from firing).

Reply:
> Or like I suggested earlier capture the \* pattern as it's own previous rule (to prevent this rule from firing).

:+1:

Reply:
Please note that I'm only updating strong rules.
Monospaced and emphasis are quite similar but first I want to make strong rules right (then apply the same rules to other formatting marks).